router-dev/Chart.yaml

```diff
@@ -10,8 +10,8 @@ details: |-
   The Deis Router is a simple program that manages Nginx and Nginx configuration for managing
   ingress to web applications in a Kubernetes cluster without requiring each to have their own
   external (to the cluster) load balancer.  Router(s) will require a single external load balancer,
-  handle all ingress, and direct all incoming traffic to appropriate services within the cluster.  
-  
+  handle all ingress, and direct all incoming traffic to appropriate services within the cluster.
```

Comment:
maybe replace "services" with "applications" to be more specific about what the router is directing traffic to?

I know this is outside the scope of this PR, so certainly not blocking